lua/KoalaVim/init.lua

```diff
@@ -44,10 +44,10 @@ function M.init()
 		DEBUG = function() end
 	end
 
-	vim.api.nvim_create_user_command('LazyRestoreLogged', function()
-		local ret = require('KoalaVim.utils.restore').restore_logged()
+	vim.api.nvim_create_user_command('LazyRestoreLogged', function(opts)
+		local ret = require('KoalaVim.utils.restore').restore_logged({ plugins = opts.fargs })
```

Comment:
If i'm not mistaken we need to parse the restore errors differently when plugins opts is passed.